packages/mobile/src/screens/edit-playlist-screen/LegacyEditPlaylistScreen.tsx

```diff
@@ -32,7 +31,7 @@ const useStyles = makeStyles(({ spacing }) => ({
   }
 }))
 
-const EditPlaylistForm = (props: FormikProps<EditPlaylistValues>) => {
+const EditPlaylistForm = (props: FormikProps<any>) => {
```

Comment:
do the edit values not work here anymore?

Reply:
They do just rather than keep the new and legacy ones around I just went with removing the ones for legacy since it'll be removed soon. Kinda silly though, I can just make a legacy one as well and add it back